doc/man/8/ceph-authtool.rst

```diff
@@ -139,7 +139,9 @@ In general, an osd capability follows the grammar::
 
         osdcap  := grant[,grant...]
         grant   := allow (match capspec | capspec match)
-        match   := [pool[=]<poolname> | object_prefix <prefix>]
+        match   := [pool[=]<poolname> | object_prefix <prefix>
+                    | tag <application-name> <key>=<value> 
+                    | namespace[=]<rados-namespace> ]
```

Comment:
We should move `namespace...` before `tag` right? Since order matters?

Reply:
Good catch!